packages/clay-alert/src/ClayAlertBase.js

```diff
@@ -113,11 +101,30 @@ class ClayAlertBase extends Component {
 	 * @private
 	 */
 	_resumeTimeout() {
-		if (this._delayTime > 0) {
-			this._startDelayTime = new Date();
+		if (this._timeToDisappear > 0) {
+			this._startedTime = new Date();
 			this._timer = setTimeout(() => {
 				this.close();
-			}, this._delayTime);
+			}, this._timeToDisappear);
+		}
+	}
+
+	/**
+	 * Sets the delayTime if passed, if it does not set the default, and starts.
+	 * @private
+	 */
+	_startTimer() {
+		if (
+			this.autoClose &&
+			(this.type === 'stripe' || this.type === 'toast')
```

Comment:
Since this is configurable and this is `ClayAlertBase` I don't think we should check the type to start the timer, only if autoClose is not null.

Reply:
Yeah, I've been thinking that this was redundant... remove it. thanks!